pkg/apis/storage/types.go

```diff
@@ -350,6 +354,14 @@ type CSINodeDriver struct {
 	TopologyKeys []string
 }
 
+// VolumeLimits is a set of resource limits for scheduling of volumes.
+type VolumeLimits struct {
```

Comment:
`VolumeNodeResources` sounds usable, however I don't like 'AttachLimit'. Non-attachable volumes may have limits too.

Reply:
The link to discussion in KEP - https://github.com/kubernetes/enhancements/pull/942#discussion_r275566976 and https://github.com/kubernetes/enhancements/pull/942#discussion_r273104306

Reply:
We can just call it `Volumes` ? Since key is called `Allocatable`. `Count` will return number of allocatable volumes on the node and `Capacity` could return total size of allocatable volumes on the node?

Reply:
The type as `Volumes` doesn't fit well in the context of the entire `storage.k8s.io` api group.